src/main/java/com/shevtsov/dao/ClientDao.java

```diff
@@ -3,5 +3,41 @@
 import com.shevtsov.domain.Client;
 
 public interface ClientDao {
+
+    /**
+     * Saves Client-object to DB.
+     * @param client the Client-object, which is saved.
+     * @return true if new client was saved successfully, or false otherwise.
+     */
     boolean saveClient(Client client);
+
+    /**
+     * Modifies client data in DB.
+     * @param id id of client, which is modified.
+     * @param newName id of client, which is modified.
+     * @param newSurname new name of client, which is modified.
+     * @param newPhone new surname of client, which is modified.
+     * @return true if client was modified successfully, or false otherwise.
+     */
+    boolean modifyClient(long id, String newName, String newSurname, String newPhone);
```

Comment:
ID is unique identifier, so it not good idea to update it

Reply:
this method searches client in storage by ID and modifies name, surname, and phon, not ID. ID is just for searching. We added client-collection yesterday, and I plan to implement real searching and modifying in this method. So the method will become more clear.